src/ifcconvert/IfcConvert.cpp

```diff
@@ -339,7 +339,9 @@ int main(int argc, char** argv) {
         ("default-material-file", new po::typed_value<path_t, char_t>(&default_material_filename),
             "Specifies a material file that describes the material object types will have"
             "if an object does not have any specified material in the IFC file.")
-		("validate", "Checks whether geometrical output conforms to the included explicit quantities.");
+		("validate", "Checks whether geometrical output conforms to the included explicit quantities.")
+		("no-wire-intersection-check", "Skip wire intersection check")
+		("strict-tolerance", "Use strict tolerance for detecting wire intersections");
```

Comment:
@aothms I'm not sure I'm capturing what this does accurately. It seems like this precision change affects more than just wire intersections . . . do you have a better description?